formatTest/typeCheckedTests/expected_output/arityConversion.re

```diff
@@ -7,7 +7,8 @@ type bcd =
 let a = TupleConstructor (1, 2);
 
 let b =
-  MultiArgumentsConstructor 1 2 [@implicit_arity];
```

Comment:
Yeah, the original code was at width 50 but didn't break, it is a bug so I fixed it.